src/HamburgerMenuButton.js

```diff
@@ -65,14 +65,15 @@ export default class HamburgerMenuButton extends Base {
     };
   }
 
-  // Pressing Space is the same as clicking the menu button.
+  // Pressing Enter or Space is the same as clicking the menu button.
```

Comment:
Hope you don't mind I went ahead and added Enter here and in Seamless button.

Reply:
Thanks